ipa-core/src/protocol/hybrid/agg.rs

```diff
@@ -193,65 +193,126 @@ pub mod test {
     const SHARD1_MKS: [u64; 7] = [12345, 12345, 34567, 34567, 78901, 78901, 78901];
     const SHARD2_MKS: [u64; 7] = [23456, 23456, 45678, 56789, 67890, 67890, 67890];
 
+    #[allow(clippy::too_many_lines)]
     fn get_records() -> Vec<TestHybridRecord> {
         let shard1_records = [
             TestHybridRecord::TestImpression {
                 match_key: SHARD1_MKS[0],
                 breakdown_key: 45,
+                key_id: 0,
+                helper_origin: "HELPER_ORIGIN".to_string(),
```

Comment:
nit: I think you can add `let helper_origin = "HELPER_ORIGIN".to_string();` at the top of the function and then you can just provide `helper_origin` here. (Similarly below with `conversion_site_domain`)

Reply:
use after move, but I can clone them at least

Reply:
ahh, yea. not quite as pretty as I was hoping